app/src/app/api/v0/inventory/[inventory]/inventoryvalue/[inventoryvalue]/route.ts

```diff
@@ -0,0 +1,27 @@
+import UserService from "@/backend/UserService";
+import { db } from "@/models";
+import { apiHandler } from "@/util/api";
+import createHttpError from "http-errors";
+import { NextResponse } from "next/server";
+
+export const GET = apiHandler(async (req, { params, session }) => {
+  const inventoryValue = await UserService.findUserInventoryValue(
+    params.inventoryvalue,
+    params.inventory,
+    session,
+  );
+
+  return NextResponse.json({ data: inventoryValue });
+});
+
+export const DELETE = apiHandler(async (req, { params, session }) => {
+  const inventoryValue = await UserService.findUserInventoryValue(
+    params.inventoryvalue,
+    params.inventory,
+    session,
+  );
+
+  await inventoryValue.destroy();
+
+  return NextResponse.json({ data: inventoryValue, deleted: true });
+});
```

Comment:
I think we have the same routes in `/api/v0/inventory/[inventory]/value/[subcategory]/route.ts`. They are just being accessed by the SubCategory ID as the identifier instead of the InventoryValue ID because when creating one (upsert) it doesn't have a set ID yet (and it's easier to search for a specific SubCategory's inventory value from the add data form etc.).

So I would prefer merging the routes or maybe just keeping the existing ones. Since will most likely also have the subCategoryId available in the frontend when calling them.